### DISCUSSION TITLE
新人进绳网有什么要注意的吗

### DISCUSSION THREAD

**Discussion by Sorasakirin** [share121/inter-knot#5233] [其他]
我刚刚才进绳网，请问怎么约绳匠？听说传奇绳匠法厄同的稿金很贵是真的吗？

**Comment by ParanoidHikari** [comment]
挂委托等绳匠来接咯，法厄同平时接菲林的委托比较多，你可以试试:)

**Comment by BaiHan2333** [comment]
我可以接家政委托

**Comment by Kazuha-hhh** [comment]
这个委托我接下了

**Comment by cuijiaqi0406** [comment]
不用，你买10几个加密的母带，之后挂绳网上，法厄同一定会接下委托，光速完成，并让你多放几个委托<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Comment by BJ-cmyk** [comment]
法厄同的号被盗了，现在是大AI时代🤓！

**Comment by nabeilishi** [comment]
这里没有绳匠，都是绳斤口牙！！！

**Comment by Fluorine-z** [comment]
我听一个朋友说现在的法厄同都是ai服务，根本没有真人

**Comment by obai-gs** [comment]
小心成为绳斤就好了
